appimage/build-appimage

```diff
@@ -46,14 +52,18 @@ _osversion=$(cat /etc/os-release | grep PRETTY_NAME | awk -F'=' '{print $2}' | s
 if [ x"$_osversion" != x"Ubuntu 14.04.5 LTS" ] ; then
```

Comment:
In line with change at https://github.com/qpdf/qpdf/pull/185/commits/568f518feb9c48b557d07923432c5fcd5b79b7c4#diff-5af93be82eec81e6f39b5d6084d6c421R52 (*"Being more general about the environment"*) this should probably also be changed to 

    if [[ ! $_osversion =~ Ubuntu\ 14.04.*\ LTS ]]; then

Reply:
...or is it just me being confused about *were* did occur *which* change in the sequence of commits?